### PR TITLE
fix: change style of partner offer notification badge

### PR DESCRIPTION
**Pull request by leamotta**
### Description

Match partner offer notification badge with [designs](https://www.figma.com/file/RHkhhJJwixhFPltqhH9ZkW/v1.5-Saves-Partner-Offers?node-id=3866%3A56653&mode=dev)
Simulator Screenshot - iPhone SE (3rd generation) - 2024-04-17 at 11 00 16
Simulator Screenshot - iPhone SE (3rd generation) - 2024-04-17 at 11 00 22
Simulator Screenshot - iPhone SE (3rd generation) - 2024-04-17 at 11 03 40

### PR Checklist

- [ ] I have tested my changes on **iOS** and **Android**.
- [x] I hid my changes behind a **[feature flag]**, or they don't need one.
- [x] I have included **screenshots** or **videos**, or I have not changed the UI.
- [x] I have added **tests**, or my changes don't require any.
- [x] I added an **[app state migration]**, or my changes do not require one.
- [x] I have documented any **follow-up work** that this PR will require, or it does not require any.
- [x] I have added a **changelog entry** below, or my changes do not require one.

### To the reviewers 👀

- [ ] I would like **at least one** of the reviewers to **run** this PR on the simulator or device.

<details><summary>Changelog updates</summary>

#### Cross-platform user-facing changes

- change style of partner offer notification badge - leamotta

#### iOS user-facing changes

-

#### Android user-facing changes

-

#### Dev changes

-

</details>

Need help with something? Have a look at our [docs], or get in touch with us.

[app state migration]: ../blob/main/docs/adding_state_migrations.md
[feature flag]: ../blob/main/docs/developing_a_feature.md
[docs]: ../blob/main/docs/README.md

### CODE REVIEW

**Comment by ArtsyOpenSource** [comment]
### This PR contains the following changes:

- Cross-platform user-facing changes (change style of partner offer notification badge - leamotta)
<p align="right">
  Generated by :no_entry_sign: <a href="https://danger.systems/js">dangerJS</a> against 20b91ab09d3aa6e6b304a641a7a67de4508c3480
</p>

**Review by narikazu — APPROVED** [review]
LGTM

**Review by narikazu — APPROVED** [review]
LGTM

File: src/app/Scenes/Activity/components/PartnerOffeBadge.tsx [hunk 1]
```diff
@@ -8,7 +8,13 @@ interface PartnerOfferBadgeProps {
...
 export const PartnerOfferBadge: React.FC<PartnerOfferBadgeProps> = ({ notificationType }) => {
   return (
-    <Text variant="xs" fontWeight="bold" color="blue100">
+    <Text
+      variant="xs"
+      color="blue100"
+      backgroundColor="blue10"
+      style={{ alignSelf: "flex-start" }}
+      px={0.5}
+    >
```
[the remark this answers is not in the document]
Comment (leamotta) [reply src/app/Scenes/Activity/components/PartnerOffeBadge.tsx:17]: Text _was_ bold. Good point with the border, I can update this here and in web

**Pull request merged**